nixos/modules/security/pam.nix

```diff
@@ -929,6 +929,22 @@ in
         '';
       };
 
+      prompt = mkOption {
+        default = null;
+        type = with types; nullOr str;
+        description = ''
+            Set individual prompt message for interactive mode. By setting this option, you can set a message to be shown shown by the <literal>interactive</literal> option. Note: Requires <literal>interactive</literal> to be set to <literal>true</literal>.
```

Comment:
I think there's an accidental repeat in the description: "shown shown" (also copied to cue-prompt's description.)

Reply:
It may also be good to add to the pam-u2f test as is done in this PR: [origin pr](https://github.com/NixOS/nixpkgs/pull/167514)

Reply:
Once these things are addressed just prompt me for re-review and I can merge